modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/test/java/com/liferay/portal/search/tuning/synonyms/web/internal/index/SynonymSetIndexReaderImplTest.java

```diff
@@ -56,44 +55,42 @@ public void setUp() throws Exception {
 	}
 
 	@Test
-	public void testFetchOptionalWithDocExistsFalse() {
+	public void testFetchWithDocExistsFalse() {
 		setUpSearchEngineAdapter(_setUpGetDocumentResponse(false));
 
 		SynonymSetIndexName synonymSetIndexName = Mockito.mock(
 			SynonymSetIndexName.class);
 
-		Optional<SynonymSet> synonymSetOptional =
-			_synonymSetIndexReaderImpl.fetchOptional(synonymSetIndexName, "id");
+		SynonymSet synonymSet = _synonymSetIndexReaderImpl.fetch(
```

Comment:
@LisztLiszt Please inline variable. Before your change, a method of the optional is called, so we need the optional to be assigned to a variable. However, after the change, you are directly asserting the variable itself, so the variable is not needed.